Adding trafficserver-5.3.2-f914e70.diff.

jrushford opened the pull request:
Add the Traffic server 5.3.2 f914e70 diff for inspection.

jianli123 commented:
could we have a high level explanation on what are the fixes and enhancements in this commit?

jrushford commented:
Parent selection was refactored to add a secondary parent list (hash ring) when using the consistent hash algorithm.  Added MSO (multi site origin) feature to Parent selection along with simple retry and dead server retry features when parents are MSO.  Updated the url_sig plugin to add signed path parameters.  merged in various upstream ATS changes.

jianli123 commented:
thank you jrushford
